### PR TITLE
support multi partition fields.

### PR DESCRIPTION
**Pull request by xianzhen**
Isuue:https://github.com/confluentinc/kafka-connect-hdfs/issues/83

### CODE REVIEW

File: src/main/java/io/confluent/connect/hdfs/partitioner/FieldPartitioner.java [hunk 1]
```diff
@@ -32,48 +32,61 @@
 
 public class FieldPartitioner implements Partitioner {
   private static final Logger log = LoggerFactory.getLogger(FieldPartitioner.class);
-  private static String fieldName;
+  private static List<String> fieldNames;
```
Comment (shikhar) [inline src/main/java/io/confluent/connect/hdfs/partitioner/FieldPartitioner.java]: I am not familiar with this code yet, but it seems weird that this field is `static`

cc @ewencp

**Comment by shikhar** [comment]
Other than question this LGTM as a nice backwards-compatible improvement

File: src/main/java/io/confluent/connect/hdfs/partitioner/FieldPartitioner.java (hunk 1 above)
Comment (xianzhen) [inline src/main/java/io/confluent/connect/hdfs/partitioner/FieldPartitioner.java]: @shikhar Agree with you. Different connectors may have different partition fields.

**Comment by shikhar** [comment]
@xianzhen whoops can you please sign the CLA here: http://clabot.confluent.io/cla

**Comment by shikhar** [comment]
@xianzhen Sorry, I merged and then realized:
- the CLA was not signed
- the tests were failing

Can you please open a fresh PR.

**Pull request merged**